src/community/roadmap/index.md

```diff
@@ -23,32 +23,38 @@ We recently:
 - released the [Summary card](https://github.com/alphagov/govuk-design-system-backlog/issues/210) variant
 - made it easier to find information about [contributing and the community](/community/)
 - improved [how contributions are prioritised](https://design-system.service.gov.uk/community/upcoming-components-patterns/)
-- added functionality for browsers to ‘find in page’ within the Accordion component`
+- added functionality for browsers to ‘find in page’ within the Accordion component
 - added support for ‘source maps’ for precompiled GOV.UK Frontend Javascript files
+- iterated the [guidance on alt text for images](/styles/images/#alt-text)
```

Comment:
```suggestion
- iterated the [guidance on images](/styles/images/)
```

Reply:
Or this is more descriptive, if we reckon it's not too controversial @stevenjmesser?

```suggestion
- iterated the [guidance on images](/styles/images/#alt-text) to help make images in government services more consistent, inclusive and accessible
```